.github/linters/.gitleaks.toml

```diff
@@ -0,0 +1,11 @@
+title = "gitleaks config"
```

Comment:
what is this file for?

Reply:
This file is to make it so the linter doesn't freak out with our example of an auth key or something like that. So that's why I have this file here now.

Reply:
hmm.. this is for the propertyToken value in our sample. What if we use a sample value 1234-sample or something like this, does it return the same error?

Reply:
Yeah, it's for the propertyToken - I'd prefer to use a more-real ID value, hence why I kept it like that.